core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.html

```diff
@@ -0,0 +1,97 @@
+<p-overlayPanel
+    #op
+    data-testId="p-overlaypanel-selector"
+    styleClass="dot-device-selector__overlayPanel"
+>
+    <div class="dot-device-selector__column" data-testId="device-selector">
+        <div class="dot-device-selector__panel">
+            <div class="dot-device-selector__header">
+                <h3 class="dot-device-selector__header-title">
+                    {{ 'editpage.device.selector.title' | dm }}
+                </h3>
+                <p-button icon="pi pi-plus" styleClass="p-button-rounded p-button-text"></p-button>
+            </div>
+            <div class="dot-device-selector__grid dot-device-selector__content">
+                <ul class="device-list">
+                    <li class="device-list-item" *ngFor="let option of defaultOptions">
+                        <button class="device-list__button" (click)="change(option)">
+                            <i [class]="option.icon"></i>
+                            <div class="device-list-item__content">
+                                <span class="device-list-item__name">{{ option.name }}</span>
+                                <span class="device-list-item__dimensions">
+                                    {{ option.cssHeight }} x {{ option.cssWidth }}
+                                </span>
+                            </div>
+                        </button>
```

Comment:
Maybe you can use a text button for this. Just be sure to use the correct classes for the label and the icons, so it uses the new global styles from this [PR](https://github.com/dotCMS/core/pull/25404)

Reply:
This is just a button for semantic porpuses.